### PR TITLE
EUI-6035

### PR DESCRIPTION
**Pull request by maliumer**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-6035

### Change description ###
Paper hearing View actuals- Display -1

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts [hunk 1]
```diff
@@ -61,6 +63,13 @@ export class HearingActualAddEditSummaryComponent implements OnInit, OnDestroy {
   public ngOnInit(): void {
     this.id = this.route.snapshot.params.id;
     this.error$ = this.hearingStore.select(fromHearingStore.getHearingActualsLastError);
+    this.hearingStore.select(fromHearingStore.getHearingValues).pipe(
+      filter((state: HearingValuesStateData) => !!state.serviceHearingValuesModel)
+    ).subscribe(
+      (state: HearingValuesStateData) => {
+        this.isPaperHearing = state.serviceHearingValuesModel.hearingChannels.indexOf('ONPPRS') > -1;
+      }
+    );
```
Comment (rayliangatos) [inline src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts:72]: it should grab the data from HearingRequestMainModel.hearingDetails.hearingChannels but not serviceHearingValuesModel. Please chat with @MateiHMCTS he is working on loading the value when the journey is started.

File: src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts [hunk 2]
```diff
@@ -4,12 +4,14 @@ import { Store } from '@ngrx/store';
 import * as moment from 'moment';
 import { combineLatest, Observable, Subscription } from 'rxjs';
 import { filter } from 'rxjs/operators';
+import { HearingConditions } from 'src/hearings/models/hearingConditions';
```
Comment (rayliangatos) [inline src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts:7]: could this be the relative path?

File: src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts [hunk 3]
```diff
@@ -55,6 +56,9 @@ export class HearingActualAddEditSummaryComponent implements OnInit, OnDestroy {
   public partyChannels: LovRefDataModel[] = [];
   public hearingDateRange: string;
   public hearingDatesAccordion = {} as { [hearingDate: string]: boolean};
+  public isPaperHearing: boolean = false;
+  private hearingRequestMainModel: HearingRequestMainModel;
+  private hearingConditions: HearingConditions;
```
Comment (rayliangatos) [inline src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts:61]: why do we nee this hearingConditions?

File: src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts [hunk 4]
```diff
@@ -112,6 +118,10 @@ export class HearingActualAddEditSummaryComponent implements OnInit, OnDestroy {
 
     if (this.hearingResult === HearingResult.CANCELLED || this.isValid()) {
       this.hearingStore.dispatch(new fromHearingStore.SubmitHearingActuals(this.id));
+      if (this.isPaperHearing) {
+        this.hearingRequestMainModel.hearingDetails.hearingChannels = [HearingChannelEnum.ONPPR.toString()];
+        this.hearingStore.dispatch(new fromHearingStore.UpdateHearingRequest(this.hearingRequestMainModel, this.hearingConditions));
```
Comment (rayliangatos) [inline src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts:123]: We actually don't need hearingConditions when we update the hearingRequest, ideally we need to use the current new fromHearingStore.SubmitHearingActuals, to call the additional service this.hearingsService.updateHearingRequest(payload), and we need to handle the errors corectly if the error is coming back either from updateHearingRequest or submitHearingActuals.

File: src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts (hunk 2 above)
Comment (maliumer) [reply src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts]: no longer required it is removed

**Review by rayliangatos — APPROVED** [review]

**Review by Kartheek-S — APPROVED** [review]

File: src/cases/store/effects/case-create.effects.ts [hunk 5]
```diff
@@ -53,7 +53,7 @@ export class CaseCreateEffects {
   applyCreatedCaseLoaded$ = this.actions$.pipe(
     ofType(fromActions.CREATED_CASE_LOADED),
     map((payload: any) => {
-       this.alertService.success(`Case #${payload.caseId} has been created.`, true);
+       this.alertService.success(`Case #${payload.caseId} has been created.`);
```
Comment (rayliangatos) [inline src/cases/store/effects/case-create.effects.ts:56]: why do we change this?

File: src/hearings/containers/request-hearing/request-hearing.page.flow.spec.ts [hunk 6]
```diff
@@ -63,7 +63,7 @@ describe('RequestHearingPageFlow', () => {
 
   it('should navigate continue', () => {
     requestHearingPageFlow.navigateAction(ACTION.CONTINUE);
-    expect(mockStore.dispatch).toHaveBeenCalledWith(new fromHearingStore.UpdateHearingRequest(hearingState.hearingRequest.hearingRequestMainModel, hearingState.hearingConditions));
+    expect(mockStore.dispatch).toHaveBeenCalledWith(new fromHearingStore.UpdateActualHearingRequest(hearingState.hearingRequest.hearingRequestMainModel));
```
Comment (rayliangatos) [inline src/hearings/containers/request-hearing/request-hearing.page.flow.spec.ts:66]: we shouldn't change this

File: src/hearings/containers/request-hearing/request-hearing.page.flow.ts [hunk 7]
```diff
@@ -48,7 +48,7 @@ export abstract class RequestHearingPageFlow {
         this.hearingStore.dispatch(new fromHearingStore.NavigateBackHearingRequest());
         break;
       case ACTION.CONTINUE:
-        this.hearingStore.dispatch(new fromHearingStore.UpdateHearingRequest(this.hearingRequestMainModel, this.hearingCondition));
+        this.hearingStore.dispatch(new fromHearingStore.UpdateActualHearingRequest(this.hearingRequestMainModel));
```
Comment (rayliangatos) [inline src/hearings/containers/request-hearing/request-hearing.page.flow.ts:51]: we shouldn't change this page flow on request hearing journey

File: src/hearings/store/actions/hearing-request.action.ts [hunk 8]
```diff
@@ -41,7 +41,7 @@ export class UpdateHearingRequest implements Action {
   public readonly type = UPDATE_HEARING_REQUEST;
 
   constructor(public hearingRequestMainModel: HearingRequestMainModel,
-              public hearingCondition: HearingConditions) {
+              public hearingCondition?: HearingConditions) {
```
Comment (rayliangatos) [inline src/hearings/store/actions/hearing-request.action.ts:44]: we shouldn't change this on request hearing journey

**Review by rayliangatos — COMMENTED** [review]
please revert the pr and make the changes.

**Pull request merged**